src/DEV-1/Outputer.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DEV_1
+{
+    class Outputer
```

Comment:
We need this class in the case, if we will have to output into other place, for example into the file.